guides/common/modules/ref_best-practices-for-patching-content-hosts.adoc

```diff
@@ -0,0 +1,15 @@
+[id="best-practices-for-patching-content-hosts_{context}"]
+= Best practices for patching content hosts
+
+* Registering hosts to {Project} requires {project-client-name}, which contains the `subscription-manager` package, `katello-host-tools` package, and their dependencies.
+For more information, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering hosts] in _{ManagingHostsDocTitle}_.
+* Use the {ProjectWebUI} to install, upgrade, and remove packages from hosts.
+You can update content hosts with job templates using SSH and Ansible.
+* Apply errata on content hosts using the {ProjectWebUI}.
+* Modify or replace job templates to add custom steps.
+This allows you to run commands or execute scripts on hosts.
+* When running bulk actions on hosts, bundle them by major operating system version, especially when upgrading packages.
+* Select *via remote execution {endash} customize first* to define the time when patches are applied to hosts when performing bulk actions.
+* When patching packages on hosts using the default package manager, {Project} receives a list of packages and repositories to recalculate applicable errata and available updates.
```

Comment:
This is a statement about what {Project} does. It's not really a best practice. Maybe you can attach it to a bullet point that is a best practice.